src/Microsoft.CodeAnalysis.VersionCheckAnalyzer/VersionCheckAnalyzer.cs

```diff
@@ -53,11 +53,11 @@ public override void Initialize(AnalysisContext context)
             {
                 compilationStartContext.RegisterCompilationEndAction(compilationContext =>
                 {
-                    if (!s_ShouldExecuteOperationAnalyzers)
+                    if (!s_ShouldExecuteAnalyzers)
                     {
                         // Version mismatch between the analyzer package '{0}' and Microsoft.CodeAnalysis '{1}'. Certain analyzers in this package will not run until the version mismatch is fixed.
-                        var arg1 = AnalyzerPackageName + "-" + AnalyzerPackageVersion;
-                        var arg2 = s_ShouldExecuteOperationAnalyzers;
+                        var arg1 = RequiredMicrosoftCodeAnalysisVersion;
+                        var arg2 = s_ShouldExecuteAnalyzers;
```

Comment:
Argument to the message seems wrong
```suggestion
                        var arg2 = s_MicrosoftCodeAnalysisVersion;
```

Reply:
Thanks, sounds good. This analyzer still doesn't work. Need to port it to reference Microsoft.CodeAnalysis 1.0 to work.